container-toolkit/release-notes.md

```diff
@@ -10,6 +10,50 @@ This document describes the new features, improvements, fixed and known issues f
 
 ______________________________________________________________________
 
+## NVIDIA Container Toolkit 1.15.0
+
+This release of the NVIDIA Container Toolkit `v1.15.0` is a major feature release.
+
+The following packages are included:
+
+- `nvidia-container-toolkit 1.15.0`
+- `libnvidia-container-tools 1.15.0`
+- `libnvidia-container1 1.15.0`
+
+   ```{note}
+   The `v1.14.0` release was the last release to include the `nvidia-container-runtime`
+   and `nvidia-docker2` packages.
+   All required functionality is included in the `nvidia-container-toolkit` package.
+   This toolkit package includes a utility to configure the Docker daemon to use the NVIDIA Container Runtime.
+   ```
+
+The following `container-toolkit` containers are included:
+
+- `nvcr.io/nvidia/k8s/container-toolkit:v1.15.0-ubi8`
+- `nvcr.io/nvidia/k8s/container-toolkit:v1.15.0-ubuntu20.04` (also as `nvcr.io/nvidia/k8s/container-toolkit:v1.15.0`)
+
+### Packaging Changes
+
+* Removed the `nvidia-container-runtime` and `nvidia-docker2` packages.
+* Removed the `libseccomp` dependency from the `nvidia-container-toolkit` package.
+* Added a `libnvidia-container-libseccomp2` meta package to allow for the `libseccomp` dependency to be satisfied by `libseccomp2` on SUSE-based systems.
+* Removed the `libnvidia-container0` jetpack dependency included for legacy Tegra-based systems.
+
+### Fixes and Features
```

Comment:
Do we also want to mention the below new features?
- Added automatic CDI spec generation when the `runtime.nvidia.com/gpu=all` device is requested by a container.
- Add discovery of the GDRCopy device (`gdrdrv`) if the `NVIDIA_GDRCOPY` environment variable of the container is set to `enabled`.